Common/Product/SharedProject/CommonConstants.cs

```diff
@@ -1,11 +1,12 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
+#if !NOVS
```

Comment:
NOVS stands for NOVS.

This file is shared between project building with and without references to VS.

(I'll split the file up, since that's much clearer)